code/src/Components/Album.css

```diff
@@ -0,0 +1,63 @@
+.albums-container {
+	display: flex;
+	flex-wrap: wrap;
+	width: 70%;
+}
+
+.album-wrapper {
+	margin-bottom: 35px;
+	width: 100%;
+	position: relative;
+	display: block;
+}
+
+.image-wrapper {
```

Comment:
<img width="761" alt="Skärmavbild 2020-10-23 kl  11 47 59" src="https://user-images.githubusercontent.com/51529201/96989192-a1f39300-1525-11eb-96c9-29cd0717b188.png">
<img width="776" alt="Skärmavbild 2020-10-23 kl  11 48 30" src="https://user-images.githubusercontent.com/51529201/96989223-ae77eb80-1525-11eb-84f2-446196f68621.png">

Reply:
thank you, this is now updated!